### PR TITLE
LPD-28956 Add additional configurations to email notifications when users are invited to work on a publication

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-28956

### CODE REVIEW

**Comment by brookedalton** [comment]
Screenshot 2024-08-13 at 5 20 49 PM
I kind of liked the "Email from Name" and "Email from Address"

File: modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/portlet/action/test/InviteUsersMVCResourceCommandTest.java [hunk 1]
```diff
@@ -206,30 +205,27 @@ private ThemeDisplay _getThemeDisplay() throws Exception {
...
 			mockLiferayResourceRequest.getHttpServletRequest());
 
-		ServiceContextThreadLocal.pushServiceContext(serviceContext);
+		ServiceContextThreadLocal.pushServiceContext(_serviceContext);
 
 		_mvcResourceCommand.serveResource(
 			mockLiferayResourceRequest, new MockLiferayResourceResponse());
 	}
 
+	@DeleteAfterTestRun
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/portlet/action/test/InviteUsersMVCResourceCommandTest.java:226]: This annotation is not needed unless DataGuard is disabled.

**Review by pei-jung — COMMENTED** [review]
@brookedalton good work! Just one minor request.

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:publications

**Comment by pei-jung** [comment]
ci:test:relevant

**Review by pei-jung — APPROVED** [review]

**Comment by brookedalton** [comment]
ci:forward

**Pull request closed without merging**